webapp/models/order_beer.py

```diff
@@ -0,0 +1,12 @@
+import logging
+
+from google.appengine.ext import ndb
+
+class OrderBeer(ndb.Model):
+    
+    order_id = ndb.IntegerProperty()
+    beer_id = ndb.IntegerProperty()
```

Comment:
I think these 2 should be `ndb.KeyProperty()` to actually have references of the objects